apps/Minecraft Java MultiMC5/install

```diff
@@ -0,0 +1,213 @@
+__os_id="${os[0]}"
+__os_desc="${os[1]}"
+__os_release="${os[2]}"
+__os_codename="${os[3]}"
+
+#to set $arch variable
+source "${DIRECTORY}/api" || error "failed to source ${DIRECTORY}/api"
+
+case "$__os_id" in
+    Raspbian|Debian)
+        case "$__os_codename" in
+            bullseye|buster|stretch|jessie)
+                "${DIRECTORY}/pkg-install" "wget apt-transport-https gnupg" "$(dirname "$0")" || exit 1
+                cd /tmp
+                rm -rf public
+                rm -rf adoptopenjdk-keyring.gpg
+                wget https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public
+                gpg --no-default-keyring --keyring ./adoptopenjdk-keyring.gpg --import public
+                gpg --no-default-keyring --keyring ./adoptopenjdk-keyring.gpg --export --output adoptopenjdk-archive-keyring.gpg 
+                rm -rf public
+                rm -rf adoptopenjdk-keyring.gpg
+                sudo mv adoptopenjdk-archive-keyring.gpg /usr/share/keyrings
+                cd ~
+                echo "deb [signed-by=/usr/share/keyrings/adoptopenjdk-archive-keyring.gpg] https://adoptopenjdk.jfrog.io/adoptopenjdk/deb $__os_codename main" | sudo tee /etc/apt/sources.list.d/adoptopenjdk.list
+                ;;
+            *)
+                error "Debian version ($__os_codename) is too old, update to debian Jessie or newer"
+                ;;
+        esac
+        # install dependencies
+        # for now don't install openjdk-8-jre on debian based 32bit installs... the repo versions (from adoptopenjdk and raspbian) are broken and don't work
+        # refer to the below bug reports for updates
+        # https://bugs.launchpad.net/raspbian/+bug/1944774
+        # https://github.com/adoptium/adoptium-support/issues/368
+
+        # download java 8 from lunar launcher repo as a workaround for armhf
+        case "$arch" in
+            "64") "${DIRECTORY}/pkg-install" "git clang gcc g++ cmake curl zlib1g-dev openjdk-8-jre openjdk-11-jre adoptopenjdk-16-hotspot-jre qtbase5-dev" "$(dirname "$0")" || exit 1 ;;
+            "32") 
+                "${DIRECTORY}/pkg-install" "subversion git clang gcc g++ cmake curl zlib1g-dev openjdk-11-jre adoptopenjdk-16-hotspot-jre qtbase5-dev" "$(dirname "$0")" || exit 1
+                mkdir -p ~/MultiMC/install/java/java-8-openjdk-armhf || exit 1
+                cd ~/MultiMC/install/java/java-8-openjdk-armhf && svn checkout https://github.com/gl91306/lunar/trunk/jre
+                cd ~
+                ;;
+            *) error "Failed to detect OS CPU architecture! Something is very wrong." ;;
+        esac
+        ;;
+    LinuxMint|Linuxmint|Ubuntu|[Nn]eon|Pop|Zorin|[eE]lementary)
+        # get the $DISTRIB_RELEASE and $DISTRIB_CODENAME first from lsb-release (for ubuntu) and then from the upstream for derivatives
+        source /etc/lsb-release
+        source /etc/upstream-release/lsb-release
+        case "$DISTRIB_CODENAME" in
+            bionic|focal|groovy)
```

Comment:
Add support for ubuntu hirsute

Just make it look like  `bionic|focal|groovy|hirsute)`

Reply:
ubuntu hirsute, impish, etc are already supported and do not need the openjdk ppa because they already include openjdk-16 in the default ubuntu repos

the get caught by the next case and the script continues

```
*)
                requiredver="18.04"
                if printf '%s\n' "$requiredver" "$DISTRIB_RELEASE" | sort -CV; then
                    echo "Skipping OpenJDK PPA, $DISTRIB_CODENAME already has openjdk-16 in the default repositories"
                    ```